unicore/ask/service/tests/test_responses_api.py

```diff
@@ -0,0 +1,143 @@
+import uuid
+
+from unicore.ask.service.tests import DBTestCase
+from unicore.ask.service.models import (
+    QuestionOption, Question, QuestionResponse)
+
+
+class ResponsesApiTestCase(DBTestCase):
+    def create_question_option(self, session=None, **attrs):
+        return self.create_model_object(QuestionOption, session, **attrs)
+
+    def create_question(self, session=None, **attrs):
+        return self.create_model_object(Question, session, **attrs)
+
+    def create_question_response(self, session=None, **attrs):
+        return self.create_model_object(QuestionResponse, session, **attrs)
+
+    def setUp(self):
+        super(ResponsesApiTestCase, self).setUp()
+
+        # Create Free Text Question
+        self.question_1 = self.create_question(
+            self.db, title='What is your name', short_name='name',
+            question_type='free_text',
+            options=[])
+        self.db.flush()
+        self.question_1_option = self.create_question_option(
+            self.db, question_id=self.question_1.uuid)
```

Comment:
A `QuestionOption` object for a `free_text` question seems redundant. I see the `QuestionResponse` model requires one though.

This is a similar model to the one used in `jmbo-polls` (I think) and it's the first one that came to my mind. But what do you think of a model where you define `option_type` on the `QuestionOption` instead of `question_type` on `Question`? So you can have a question with 3 `multiple_choice` options and one `free_text` option, e.g. pick option A, B or C or fill out an "Other" field. This model might introduce other complications though. Just throwing it out there.

PS. Sorry I didn't bring this up in the previous PR.